home/forms/search.py

```diff
@@ -29,16 +30,17 @@ def get_sort_choices():
     ]
 
 
-def get_classification_choices():
-    return [
-        ("OFFICIAL", "Official"),
-    ]
-
-
 def get_where_to_access_choices():
     return [("analytical_platform", "Analytical Platform")]
 
 
+def get_entity_types():
+    return [
+        (entity.name, entity.name.replace("_", " ").lower().title())
+        for entity in ResultType if entity.name != "GLOSSARY_TERM"
+    ]
```

Comment:
could we sort these maybe? Something like
```suggestion
    entities = [
        (entity.name, entity.name.replace("_", " ").lower().title())
        for entity in ResultType
        if entity.name != "GLOSSARY_TERM"
    ]
    entities.sort(key=lambda x: x[0])
    return entities